R/redist-package.R

```diff
@@ -10,7 +10,7 @@
 #' @importFrom foreach foreach %do% %dopar%
 #' @importFrom doRNG %dorng%
 #' @importFrom grDevices dev.off pdf
-#' @importFrom stats median runif quantile qnorm IQR optim sd qt var na.omit
+#' @importFrom stats sd var na.omit median runif quantile qnorm IQR optim splinefun qt
```

Comment:
One of these days we should clean up this file. No action items on this file for this PR but just a note to think about.